nni/compression/pytorch/speedup/error_code.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# Error Code of the speedup
+class SpeedupError(Exception):
+    def __init__(self, msg):
+        self.msg = msg
+
+    def __str__(self):
+        return str(self.msg)
+
+class EmptyLayerError(SpeedupError):
+    def __init__(self):
+        super(EmptyLayerError, self).__init__("Pruning a Layer to empty is not legal")
+
+class ShapeMisMatchError(SpeedupError):
+    def __init__(self):
+        super(ShapeMisMatchError, self).__init__("Shape mismatch!")
```

Comment:
maybe we can also add layer name in the error message for debugging?

Reply:
Good suggestion, but I cannot get the `op_name` in the module replace functions under current interfaces. To keep the replacement function interface clean, I suggest to just report the error type now.